frontend/src/components/DietPlan/StandardMealItem.tsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import { StyleSheet, Text, View } from "react-native";
+import NativeIcon from "../Icon/NativeIcon";
+import { Colors } from "@/constants/Colors";
+import { MaterialCommunityIconsNames } from "@/types/iconTypes";
+
+interface StandardMealItemProps {
+  quantity: number;
+  iconName: MaterialCommunityIconsNames;
+}
+
+const StandardMealItem: React.FC<StandardMealItemProps> = ({ quantity, iconName }) => {
+  return (
+    <View style={styles.mealItems}>
+      <NativeIcon
+        library="MaterialCommunityIcons"
+        name={iconName}
+        size={20}
+        color={Colors.primary}
+      />
+      <Text style={styles.mealItemsText}>חלבונים: {quantity}</Text>
+    </View>
+  );
+};
+
+const styles = StyleSheet.create({
+  mealItems: {
+    display: "flex",
```

Comment:
I wanted to mention this in our meeting but all View elements always start with flex flexDirection:flex-col as default. So you  only need to change the flexDirection